### PR TITLE
add pointers for parallels' users.

### PR DESCRIPTION
**Pull request by AntonioMeireles**
hi all, 

this shouldn't hurt, and would help a few ppl and close a few tickets. (and i'm commited to keep updated boxes at Atlas to match the official ones)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,40 +19,57 @@ cd coreos-vagrant
...
-vagrant up
-vagrant ssh
-```
+  ```
+  vagrant up
+  vagrant ssh
+  ```
 
-**VMware Provider**
+ - **VMware Provider**
```
Comment (bcwaldon) [inline README.md]: This is getting bulleted under the Virtualbox Provider - it shouldn't be, right?

File: README.md [hunk 2]
```diff
@@ -19,40 +19,57 @@ cd coreos-vagrant
...
-vagrant up --provider vmware_workstation
-vagrant ssh
-```
 
 ``vagrant up`` triggers vagrant to download the CoreOS image (if necessary) and (re)launch the instance
 
 ``vagrant ssh`` connects you to the virtual machine.
 Configuration is stored in the directory so you can always return to this machine by executing vagrant ssh from the directory where the Vagrantfile was located.
 
+
```
Comment (bcwaldon) [inline README.md]: Unnecessary newlines

File: Vagrantfile [hunk 3]
```diff
@@ -54,6 +54,10 @@ Vagrant.configure("2") do |config|
       override.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant_vmware_fusion.json" % $update_channel
     end
   end
+  config.vm.provider :parallels do |vb, override|
```
Comment (bcwaldon) [inline Vagrantfile]: We need to generate parallels boxes and host them on the normal storage site (release.core-os.net) as a part of our normal build process. Maybe @marineam or @crawford can help with that.

File: Vagrantfile [hunk 4]
```diff
@@ -110,6 +118,11 @@ Vagrant.configure("2") do |config|
         vb.memory = vm_memory
         vb.cpus = vm_cpus
       end
+      config.vm.provider :parallels do |vb|
+        vb.memory = vb_memory
```
Comment (bcwaldon) [inline Vagrantfile]: I believe these should be `vm_memory` and `vm_cpus`

**Comment by bcwaldon** [comment]
@AntonioMeireles Please update your commit message per CONTRIBUTING.md

File: Vagrantfile (hunk 3 above)
Comment (AntonioMeireles) [inline Vagrantfile]: yes, the ones i 'm pointing are just a stop gap measure, until that happens :smile:

**Comment by AntonioMeireles** [comment]
@bcwaldon  i _think_ latest commit handles all your points. thanxs!

**Comment by bcwaldon** [comment]
@AntonioMeireles ok, so to be clear - I'm not willing to point users of this project to `https://vagrantcloud.com/AntonioMeireles/coreos-%s`. Please work with @crawford to make the necessary changes in our build process so we can publish parallels boxes.

**Comment by AntonioMeireles** [comment]
@crawford ping. per @bcwaldon comments above where and how can one help ?  (fwiw the 'hack' i inherited from @YungSang is [here](https://github.com/AntonioMeireles/coreos-packer))

**Comment by jgkim** [comment]
@AntonioMeireles To build Parallels images using the CoreOS SDK, a template for Parallels configuration `config.pvs` is needed (cf. https://github.com/coreos/bugs/issues/5), but as far as I know, `config.pvs` should contain every setting available, making it quite verbose. Maybe @legal90 could help solve this issue.

**Comment by legal90** [comment]
I do not recommend you to edit `config.pvs` manually because Parallels Desktop rewrite this file automatically. It is better to use `prlctl` CLI utility to configure virtual machine.

**Comment by legal90** [comment]
@AntonioMeireles, @jgkim, @crawford 
Could you please explain, is there some stoppers for this pull-request to be merged? May be I can help somehow.

P.s. I work at Parallels and maintain Parallels provider for Vagrant (https://github.com/Parallels/vagrant-parallels).

**Comment by AntonioMeireles** [comment]
@legal90 - "my way" is a plain hack( and obviously apocriphal) and i fully understand CoreOS' reasoning on not wanting to endorse in any sort of way 3rd party hacks in their official documentation. OTOH to do things right one would have to integrate Parallels builds in the CoreOS' internal release building/validation pipeline which is obviously a bit complicated to do for anyone not in the within... (and the ~10k downloads my hack hadn't ben soo far a good reason so far. dunno if because statistically irrelevant or if because they just work ence people not noisy enough about this)

**Comment by jgkim** [comment]
@legal90 For this pull-request to be merged, we need a way to build Parallels images without Parallels, including `prlctl` CLI utility. As far as I know, you can build a hard disk image for Parallels by using `qemu-img`, but without a template for Parallels configuration `config.pvs`, you cannot build a VM image for Parallels. Look into CoreOS Developer SDK, especially https://github.com/coreos/scripts/blob/master/build_library/vm_image_util.sh.

**Comment by crawford** [comment]
Closing due to inactivity.

**Pull request closed without merging**